lua/nui/popup/border.lua

```diff
@@ -391,25 +391,30 @@ function Border:reposition()
     props.position = calculate_position(self)
   end
 
-  local win_config = {
-    row = props.position.row,
-    col = props.position.col,
-    relative = self.popup.win_config.relative,
-  }
+  self.win_config.relative = self.popup.win_config.relative
 
-  if self.win_config.bufpos then
-    win_config.bufpos = self.win_config.bufpos
+  if self.popup.win_config.bufpos then
+    self.win_config.bufpos = self.popup.win_config.bufpos
+    self.win_config.win = nil
   end
 
-  if self.win_config.win then
-    win_config.win = self.win_config.win
+  if self.popup.win_config.win then
+    self.win_config = self.popup.win_config.win
```

Comment:
should be:
```lua
self.win_config.win = self.popup.win_config.win
```

Reply:
Let me know when you're finished working on this 😃

Reply:
@MunifTanjim finished!

Reply:
Not sure why stylua CI is failing, I formatted all files with the .stylua config in the repo.

Reply:
No worries. I'll squash the commits into a single commit and format again.

By the way, the commits don't have your right email address associated with your github account. Is that intentional? Or do you want to fix that? 🤔

Reply:
That's my bad, I switched OS and forgot to update it. Thank you for pointing that out. 
I've updated it, `2744926` is with my github. I'm guessing when you squash you can use the author/email from that?

Reply:
> I'm guessing when you squash you can use the author/email from that?

Yep. Thanks! 😃